src/components/LibraryContainer.tsx

```diff
@@ -20,12 +20,12 @@ export interface LibraryContainerProps {
 }
 
 export interface LibraryContainerStates {
-    inSearchMode: boolean,
-    searchText: string,
-    selectedCategories: string[],
-    structured: boolean,
-    detailed: boolean
-    showItemSummary: boolean
+    inSearchMode?: boolean,
```

Comment:
Yah, not entirely sure why are these changes made, @pboyer, there's no [documentation](https://github.com/DynamoDS/Dynamo/wiki/Incremental-Pull-Request-Template) on the PR so you need to buy us donuts!